app/main.py

```diff
@@ -391,24 +402,27 @@ def get_discover_path():
 # important to keep the relative <path> for accessing
 # other required files.
 @app.route("/s3-resource/<path:path>")
-def direct_download_url(path):
+def direct_download_url(path, bucket_name=Config.S3_BUCKET_NAME):
```

Comment:
Not sure why the bucket_name appears in the parameter list, it is not consistent with any other similar API.

Reply:
It is not consistent as it is called by directly by simulation_ui_file as well thus the parameter.

Reply:
I will add the parameter for all the other similar api.